bootstrap-stage1-chez.sh

```diff
@@ -20,5 +20,14 @@ ${SCHEME} --script ../bootstrap/compile.ss
 # Put the result in the usual place where the target goes
 mkdir -p ../build/exec
 mkdir -p ../build/exec/idris2_app
-install ../bootstrap/idris2-boot.sh ../build/exec/idris2
+
+# we "install" bootstrap/idris2-boot.sh as build/exec/idris2
+# but with the SCHEME var that we already know at this time
+# baked into it.
+echo '#!/bin/sh' >../build/exec/idris2
+echo "SCHEME=${SCHEME}" >>../build/exec/idris2
```

Comment:
Will this work if there is a space or a character that needs escaping in the `SCHEME`  path?

Reply:
I think not. Good catch.